src/signals/incident/definitions/wizard-step-2-vulaan.js

```diff
@@ -1,4 +1,8 @@
+import { Validators } from 'react-reactive-form';
+import memoize from 'lodash/memoize';
```

Comment:
Did you come across https://dev.to/nioufe/you-should-not-use-lodash-for-memoization-3441? Maybe we'd better replace if with `defaultMemoize` from `reselect`. A package we're also using in this application.

Reply:
I don't see how I can create a custom cache key with `defaultMemoize` from reselect, like I do with `memoize` from lodash. Memoizing on the `questions` argument doesn't work since it changes all the time. I need to check for the combination of category/subcategory.
Besides that, I don't expect a memory leak since no unique IDs are being used for instance. There will only ever be a very limited set of category/subcategory pairs.